src/main/java/nextstep/ladder/domain/user/UserName.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.ladder.domain.user;
+
+public class UserName {
+    private String userName;
+    private final int USER_NAME_MAXIMUM = 5;
```

Comment:
```suggestion
    private final int USER_NAME_MAXIMUM = 5;
    
    private String userName;
```

클래스 구현 순서는 아래와 같습니다! 

```
class A {
    상수(static final) 또는 클래스 변수
    인스턴스 변수
    생성자
    팩토리 메소드
    메소드
    기본 메소드 (equals, hashCode, toString)
}
```